tests/test_activation.py

```diff
@@ -150,25 +150,25 @@ def test_activation_matrix_large():
     def time_test(func, of=""):  # pylint: disable=invalid-name
         def dec_func(*args, **kwargs):
             print("start test '{}'".format(of))
-            start = time.clock()
+            start = time.time()
             res = func(*args, **kwargs)
-            end = time.clock()
+            end = time.time()
             print("finished test '{}'".format(of))
             print("  duration: {:.3f}s".format(end - start))
             print("")
             return res
         return dec_func
 
     nn = 2000
-    n_cues = 10*nn
+    n_cues = 10 * nn
     n_outcomes = nn
-    n_events = 10*nn
+    n_events = 10 * nn
     n_cues_per_event = 30
-    weight_mat = np.random.rand(n_cues, n_outcomes)
+    weight_mat = np.random.rand(n_outcomes, n_cues)
     cues = ['c'+str(i) for i in range(n_cues)]
```

Comment:
Not really part of your proposed changes, but we could change the `i` here to `ii` as well